tomviz/resources/defaultcolormaps.json

```diff
@@ -394,5 +394,14 @@
             0.746032, 1, 0, 0,
             1, 0.5, 0, 0
         ]
-    }
+    },
+    {
+        "Name": "Grayscale",
+        "ColorSpace": "RGB",
+        "NanColor": [ 1, 0, 0],
+        "RGBPoints": [
+          0, 0, 0, 0,
```

Comment:
Can you match the indentation used above, minor nitpick, otherwise it looks good. Thanks

Reply:
Sorry about that, fixed!